src/maintainer/conda_forge_yml.rst

```diff
@@ -537,3 +555,22 @@ Currently only:
 
     win:
       enabled: False
+
+.. _woodpecker:
+
+woodpecker
+----------
+This dictates the behavior of the woodpecker CI service (a fork of Dune).
+https://github.com/woodpecker-ci/woodpecker
+It is a mapping for Woodpecker-specific configuration options.
+For example:
+
+.. code-block:: yaml
+
+    woodpecker:
+      # Flag for uploading to anaconda. (common setting)
+      upload_packages: True
+      # woodpecker specific settings
+      yum_specific_setup:
+      build_setup:
+
```

Comment:
I am looking for some guidance on `woodpecker`.
I suspect it is similar to `dune.io` but that seems to have either been removed or never provided.

Reply:
* https://github.com/conda-forge/conda-smithy/blob/bb2bb5aae536acd4867f63584ec0dc5a466db164/conda_smithy/templates/woodpecker.yml.tmpl
* https://github.com/conda-forge/conda-smithy/blob/bb2bb5aae536acd4867f63584ec0dc5a466db164/conda_smithy/configure_feedstock.py
These did not offer much illumination.